### ISSUE TITLE
The Windows package doesn't support "generic" windows (win-x64, win-x86)

### ISSUE THREAD

**Issue by Petermarcu** [ericsink/SQLitePCL.raw#194]
This package https://www.nuget.org/packages/SQLitePCLRaw.lib.e_sqlite3.v110_xp/

Should have a win-x64 and win-x86 folder in it as to support people publishing applications using those runtime identifiers. The guidance for .NET Core 2.0 forward is to use those by default. Likely all that needs to happen to fix this is to move the binaries from the win7-* folder to win-*.

You can see an example of how this manifests in this issue: https://github.com/ElectronNET/Electron.NET/issues/82

**Comment by ericsink** [comment]
See previous discussion on #184.

Bottom line, yes, this should happen.  The only reason it has not is because my crazy build system (which I have come to loathe) makes this ultra-minor change unnecessarily risky, and I haven't gotten back to it yet.

But I do plan to have this change in 1.1.10, which should happen in the next few weeks.

**Comment by Petermarcu** [comment]
Ok, I don't have enough context on your build system. In general, this should just put things in a win-* folder instead of a win7-* folder. Its a compatible change from NuGet's perspective but if there are other places where there are assumptions on this, then there could be side effects.

**Comment by ericsink** [comment]
SQLitePCL.raw pre-release version 1.1.10-pre20180223200113 has been pushed to nuget.  Starting with this release, the RID for these "generic" Windows builds begins with win- instead of win7-.  Let me know if you find any problems.

**Comment by tsublette** [comment]
I don't see SQLitePCL.raw pre-release version 1.1.10-pre20180223200113 on the nuget site. Only 0.9.3 https://www.nuget.org/packages/SQLitePCL.raw/

What am I missing?

**Comment by tsublette** [comment]
never mind ... read the read me... Thanks :)

**Comment by tsublette** [comment]
I have included the preview version of SQLitePCLRaw.core and SQLitePCLRaw.bundle_green.

```
  <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.0" />
    <PackageReference Include="Microsoft.AspNetCore.SignalR" Version="1.0.0-alpha2-final" />
    <PackageReference Include="Serilog.AspNetCore" Version="2.1.0" />
    <PackageReference Include="Serilog.Sinks.ColoredConsole" Version="3.0.1" />
    <PackageReference Include="SQLitePCLRaw.bundle_green" Version="1.1.10-pre20180223200113" />
    <PackageReference Include="SQLitePCLRaw.core" Version="1.1.10-pre20180223200113" />
  </ItemGroup>
```
and I am building the win-x64 runtime using this command

```
dotnet publish -c Release -r win-x64 -o /app/binaries/win-x64
```
and the dll - e_sqlite3.dll - is still missing from the publish folder.

Can you give me some insight into what I am doing wrong?

**Comment by ericsink** [comment]
I'm not sure.

This bat file:

    mkdir t194
    cd t194
    dotnet new console
    dotnet add package -v 1.1.10-pre20180223200113 SQLitePCLRaw.bundle_green
    dotnet publish -c Release -r win-x64
    dir bin\Release\netcoreapp2.0\win-x64\publish\e*.dll

shows me

     Directory of C:\Users\eric\dev\t194\bin\Release\netcoreapp2.0\win-x64\publish

    02/23/2018  08:03 PM         1,238,016 e_sqlite3.dll
                   1 File(s)      1,238,016 bytes

but if I change the bat file to use `-v 1.1.9` in the `dotnet add package` line, the `dir` command finds nothing.

Maybe try those same commands on your system and see if you get the same result?

**Comment by tsublette** [comment]
I removed the reference to 
```
<PackageReference Include="SQLitePCLRaw.core" Version="1.1.10-pre20180223200113" />
```
 and kept the reference to SQLitePCLRaw.bundle_green and it worked.

Thanks for your quick response

**Comment by ericsink** [comment]
This should be fixed in 1.1.10, which is now on nuget.org
